json_object_private.h

```diff
@@ -20,7 +20,6 @@
 extern "C" {
 #endif
 
-struct json_object;
```

Comment:
This is here to allow json_object_private.h to be included on its own.  If this is removed it's possible to result in a `warning: ‘struct json_object’ declared inside parameter list `